streams/examples/main/java/org/apache/kafka/streams/examples/wordcount/WordCountJob.java

```diff
@@ -79,23 +79,9 @@ public static void main(String[] args) throws Exception {
                         return new KeyValue<String, String>(value, value);
                     }
                 })
-                .countByKey(UnlimitedWindows.of("Counts").startOn(0L),
-                        stringSerializer, longSerializer,
-                        stringDeserializer, longDeserializer)
-                .toStream()
-                .map(new KeyValueMapper<Windowed<String>, Long, KeyValue<String, JsonNode>>() {
-                    @Override
-                    public KeyValue<String, JsonNode> apply(Windowed<String> key, Long value) {
-                        ObjectNode jNode = JsonNodeFactory.instance.objectNode();
-
-                        jNode.put("word", key.value())
-                             .put("count", value);
-
-                        return new KeyValue<String, JsonNode>(null, jNode);
-                    }
-                });
+                .countByKey(stringSerializer, longSerializer, stringDeserializer, longDeserializer, "Counts");
```

Comment:
Weird.  Using the latest Kafka `trunk` code, this call to `countByKey` doesn't even compile for me.